.npmignore

```diff
@@ -1,10 +1,13 @@
-# test
+# source files
```

Comment:
Can you run `npm pack` and validate that all still looks and works well?

Reply:
I've validated that `npm pack` outputs both `lib` and `src`.